README.md

```diff
@@ -17,13 +18,40 @@
 * Operates on top of Selenium.
 * The Git for your GUI.
 
+
+### Easy creation and maintenance
+
+Instead of manually defining individual aspects that you want to check, check everything at once. 
```

Comment:
Why do you create line breaks at a random length? I think we should either avoid hard wraps at all (since most editors and GitHub insert soft wraps automatically, see e.g. line 41) or insert hard wraps at a specific length (e.g. 80, 100, … characters).

Reply:
Avoiding hard wraps...